smalltalksrc/VMMakerTests/VMObjectLayoutTests.class.st

```diff
@@ -253,6 +265,30 @@ VMObjectLayoutTests >> testObjectWithMoreThan254SlotsHasTwoHeaders [
 		self assert: numSlots equals: slots ]
 ]
 
+{ #category : #'tests - integerValues' }
+VMObjectLayoutTests >> testPositiveIntegerBorderCase1 [
+"Test of the border case when int = 2r000111111 ... . Regarding the cCode implementation, the sign bit can be lost if this test fails. The maxCInteger and numSmallIntegerTagBits guaranties the portability of the test on 32 and 64 bit computer. "
```

Comment:
Please indent the comment.
What means border case, the max value possibly encoded on ...

guaranties -> guarantees